src/components/ExportCombinedPopUp.vue

```diff
@@ -89,14 +86,15 @@ export default {
     }
   },
   methods: {
-    async exportMap () {
-      if (!this.exportName && !this.formErrors.includes("Schaal is verplicht")) {
-        this.formErrors.push("Schaal is verplicht")
-      } else if (this.exportName) {
-        this.formErrors = []
+    validateForm () {
+      if (!this.exportScale) {
+        this.hasError = true
+      } else if (this.exportScale) {
+        this.hasError = false
```

Comment:
In essence what this says is "if true, else if false". I also noticed that the field for naming this export is gone. Is this intentional?

Reply:
> also noticed that the field for naming this export is gone. Is this intentional?

In the combined export, naming the export was never implemented